components/Faq.vue

```diff
@@ -0,0 +1,15 @@
+<template>
+  <div>
+      <section-title :path="`${qanda}.question`" :margin-bottom="false" />
+      <paragraphs :path="`${qanda}.answer`" :marginBottom="false" />
```

Comment:
Indentation is 6 spaces instead of 4 here and `marginBottom` should be `margin-bottom`.
As this component has no special behaviors I think we can use the `paragraphs` component with an array instead of adding another component.

Reply:
Done